{
  "type": "fail",
  "message": "No PR with this number exists, ({\"data\":{\"repository\":{\"__typename\":\"Repository\",\"pullRequest\":null}},\"loading\":false,\"networkStatus\":7,\"errors\":[{\"type\":\"NOT_FOUND\",\"path\":[\"repository\",\"pullRequest\"],\"locations\":[{\"line\":3,\"column\":5}],\"message\":\"Could not resolve to a PullRequest with the number of 49500.\"}]})"
}